File: swagger-client/scala-akka-http-server/src/main/scala/io/swagger/server/model/EmailAddressName.scala
package io.swagger.server.model


/**
 * = EmailAddressName =
 *
 * An email contact with an optional display name.  Used in structured address fields (`from`, `to`, `cc`, `bcc`, `replyto`) in `SendMailAdv`.
 *
 * @param email The email address. for example: ''user@domain.com''
 * @param name Optional display name shown to recipients (e.g. in the `From:` header). for example: ''John Smith''
 */
case class EmailAddressName (
  email: String,
  name: Option[String]
)

File: openapi-client/javascript-closure-angular/API/Client/EmailAddressName.js
goog.provide('API.Client.EmailAddressName');

/**
 * An email contact with an optional display name.  Used in structured address fields (`from`, `to`, `cc`, `bcc`, `replyto`) in `SendMailAdv`.
 * @record
 */
API.Client.EmailAddressName = function() {}

/**
 * The email address.
 * @type {!string}
 * @export
 */
API.Client.EmailAddressName.prototype.email;

/**
 * Optional display name shown to recipients (e.g. in the `From:` header).
 * @type {!string}
 * @export
 */
API.Client.EmailAddressName.prototype.name;

